sdk/daml-lf/interpreter/src/main/scala/com/daml/lf/speedy/SBuiltinFun.scala

```diff
@@ -2085,7 +2085,7 @@ private[lf] object SBuiltinFun {
 
   private def extractContractInfo(
       tmplId2TxVersion: TypeConName => TransactionVersion,
-      tmplId2PackageName: TypeConName => PackageName,
+      tmplId2PackageNameVersion: TypeConName => (PackageName, Option[PackageVersion]),
```

Comment:
And in a subsequent PR we make it mandatory, after Canton forwards it

Reply:
Unfortunately we will have to keep it optional as long LF 2.1 is alive. 
I home we will be able to kill this LF version before mainet is stable and make the field mandatory